earth2mip/schema.py

```diff
@@ -98,7 +98,12 @@ class EnsembleRun(pydantic.BaseModel):
         weather_event (optional): The weather event to use for the forecast (alternative to `forecast_name`).
         output_dir (optional): The directory to save the output files in (alternative to `output_path`).
         output_path (optional): The path to the output file (alternative to `output_dir`).
-        restart_frequency: if provided save at end and at the specified frequency. 0 = only save at end.
+        model_registry (optional): The path to the model registry (alternative to the default, $HOME/.cache/earth2mip/models
```

Comment:
Just fixing a missing parenthesis
```suggestion
        model_registry (optional): The path to the model registry (alternative to the default, $HOME/.cache/earth2mip/models)
```

Reply:
thanks for catching this, fixed